lib/ast-converter.js

```diff
@@ -293,11 +293,11 @@ function getTokenType(token) {
         }
     }
 
-    if (token.kind >= 68 && token.kind <= 112) {
+    if (token.kind >= 70 && token.kind <= 114) {
```

Comment:
Rather than just hardcoding the numbers, I think we should try to make use of the token boundary values: https://github.com/Microsoft/TypeScript/commit/b24c3a7aff3bf6e50a43256c72ae7a6851e35673#diff-29796169c2bd635765a9f574727f77f3R310

As far as I can tell, using `SyntaxKind.FirstKeyword` and `SyntaxKind.LastFutureReservedWord` here might inoculate us from this problem in the future.